gradle/verification-metadata.xml

```diff
@@ -2112,17 +2112,29 @@
             <sha256 value="0bdd11c226ff836057f753f8426b4f77efc5fdba96f55692eceea82656c93ac0" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="common-custom-user-data-gradle-plugin" version="1.6.5">
```

Comment:
is there somewhere where the old version can be removed?

Reply:
They're using it in buildSrc and I didn't bump it up there as impact is more complicated to estimate
https://ge.solutions-team.gradle.com/s/kbc552lnvsyqq/build-dependencies?dependencies=gradle&expandAll&toggled=W1swLDFdLFswLDFdXQ

Reply:
If they are using it in buildSrc then we shouldn't need to declare the version name twice.